tests/cases/fourslash/completionInIncompleteCallExpression.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path='fourslash.ts'/>
+
+////var array = [1, 2, 4]
+////function a4(x, y, z) { }
+////a4(...<crash>/**/

```

Comment:
In the issue, it is to indicate that this will be a crash. Though it is still a nice test case to have :)